test/operations.jl

```diff
@@ -0,0 +1,30 @@
+if VERSION ≥ v"1.3"
+@testset "|> syntax for pipelines" begin
+    Random.seed!(142)
+    @load RidgeRegressor
+    @load KNNRegressor
+    X = MLJBase.table(randn(500, 5))
+    y = abs.(randn(500))
+    train, test = partition(eachindex(y), 0.7)
+
+    # Feeding data directly to a supervised model
+    knn = KNNRegressor(K=10)
+    ŷ   = (X, y) |> knn
+    fit!(ŷ, rows=train)
+
+    # Describing a full pipeline using |> syntax.
+    Xs, ys = source.((X, y))
```

Comment:
This will work here but, just FYI, if we want to export the network as stand-alone model, you now need to do
`Xs = source(X)` (or `Xs = source(X, kind=:input)`) and `ys = source(y, kind=:target)`.